code/game/gamemodes/cult/runes.dm

```diff
@@ -360,46 +376,41 @@ var/list/teleport_runes = list()
 		rune_in_use = 0
 		return*/
 	if(((ishuman(offering) || isrobot(offering)) && offering.stat != DEAD) || is_sacrifice_target(offering.mind)) //Requires three people to sacrifice living targets
-		var/cultists_nearby = 1
-		for(var/mob/living/M in range(1,src))
-			if(iscultist(M) && M != user)
-				cultists_nearby++
-				M.say(invocation)
-		if(cultists_nearby < 3)
-			user << "<span class='cultitalic'>[offering] is too greatly linked to the world! You need three acolytes!</span>"
+		if(invokers.len < 3)
+			for(var/M in invokers)
+				M << "<span class='cultitalic'>[offering] is too greatly linked to the world! You need three acolytes!</span>"
 			fail_invoke()
 			log_game("Sacrifice rune failed - not enough acolytes and target is living")
 			rune_in_use = 0
 			return
 	visible_message("<span class='warning'>[src] pulses blood red!</span>")
 	color = rgb(126, 23, 23)
-	spawn(5)
-		color = initial(color)
-	sac(offering)
+	..()
+	sac(invokers, offering)
+	color = initial(color)
```

Comment:
..() doesn't change color, it only changes alpha and transform. It'd conflict with fail_invoke(), but if we're calling ..() in invoke we aren't going to call fail_invoke